docs/DeveloperGuide.adoc

```diff
@@ -806,9 +806,36 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |edit the detail of a person
+ |easily make change to their detail when they updated their contact
+
+|`* * *` |user |delete a person |remove entries that I no longer need
```

Comment:
teacher| delete the contact of a student | remove contacts that I no longer need